lib/apiservers/engine/backends/convert/stats.go

```diff
@@ -316,3 +417,31 @@ func (cs *ContainerStats) currentCPU() {
 	dockerCPU.CPUUsage.TotalUsage += cs.preTotalMhz
 	cs.curDockerStat.CPUStats = dockerCPU
 }
+
+// diskMinor will parse the vSphere provided disk name and return the minor id of
+// the disk device.  The func assumes that minor identifiers are multiples of
+// 16 (0,16,32,48,etc) and that the containerVM has 10 or less disks
+func diskMinor(containerID string, name string) uint64 {
+	// retrieve the last char
+	last := string(name[len(name)-1])
+	// the int suffix should always be last
+	minor, err := strconv.Atoi(last)
```

Comment:
when creating this I apparently switched controller maximum w/disk maximum...so this was coded to having fewer than 10 disks.  I'll need to parse the string differently.  FTR the names look like this: `scsi0:0`